packages/backend/server/schema.prisma

```diff
@@ -444,7 +444,7 @@ model AiContextEmbedding {
   // a file can be divided into multiple chunks and embedded separately.
   chunk     Int                        @db.Integer
   content   String                     @db.VarChar
-  embedding Unsupported("vector(512)")
+  embedding Unsupported("vector(1024)")
```

Comment:
need a migration?

Reply:
Not needed, this is just a definition with no actual effect, canary has been manually fixed